source/_integrations/template.markdown

```diff
@@ -330,6 +330,36 @@ sensor:
 
 Useful entities to choose might be `sensor.date` which update once per day or `sensor.time` which updates once per minute.
 
+Sometimes it is reasonable to convert an entity-less template that operates with time into one with entities to get your sensor value updated automatically:
+
+{% raw %}
+
+```yaml
+# Original entity-less template sensor that WON'T be updated automatically
+- platform: template
+    sensors:
+      house_asleep:
+        value_template: >-
+          {{ is_state('binary_sensor.anybody_home', 'off') and (now().hour|float > 22 or now().hour|float < 6)) }}
+```
+
+{% endraw %}
+
+{% raw %}
+
+```yaml
+# Converted template sensor that uses sensor.time entity and WILL BE be updated automatically on every sensor.time update (i.e every minute)
```

Comment:
This line is documentation, not code comments.

Reply:
it's gone now